src/main/java/seedu/address/model/flashcard/Flashcard.java

```diff
@@ -114,16 +114,16 @@ public String toString() {
         final StringBuilder builder = new StringBuilder();
         builder.append("Question: ")
                 .append(getQuestion())
-                .append("; Answer: ")
+                .append("\nAnswer: ")
                 .append(getAnswer())
-                .append("; Category: ")
+                .append("\nCategory: ")
                 .append(getCategory())
-                .append("; Priority: ")
+                .append("\nPriority: ")
                 .append(getPriority());
 
         Set<Tag> tags = getTags();
         if (!tags.isEmpty()) {
-            builder.append("; Tags: ");
+            builder.append("\nTags: ");
```

Comment:
Good change! The addition of new lines after each component of the flashcard will help to improve readability.